package.sh

```diff
@@ -8,6 +8,33 @@
 # Exit on error.
 set -e
 
+postin_message()
+{
+	cat << 'END_HEREDOC'
```

Comment:
nit, usually EOF is a short-n-sweet delimiter used and no need to put quotes,
also L21 MUST have the delimiter as the only word, 
so

cat << EOF
...
...
EOF

note that the line must start with EOF w/o spaces/tabs before that